src/segments/helm.go

```diff
@@ -5,6 +5,11 @@ import (
 	"github.com/jandedobbeleer/oh-my-posh/src/properties"
 )
 
+// List of filenames to enable when display_mode is "files"
+var helmDisplayFiles = []string{
```

Comment:
I prefer these are kept inline rather than defined globally here, there's no real use-case to expose that